tests/foreman/cli/test_host_collection.py

```diff
@@ -547,26 +625,77 @@ def test_positive_host_collection_host_pagination(self):
 
         :CaseLevel: Integration
         """
-        host_collection = self._new_host_collection({
-            'name': gen_string('alpha', 15)
+        host_collection = make_host_collection({
+            'organization-id': self.organization['id'],
         })
         host_ids = ','.join(
             self._make_fake_host_helper()['id'] for _ in range(2)
         )
         HostCollection.add_host({
             u'host-ids': host_ids,
             u'id': host_collection['id'],
-            u'organization-id': self.org['id'],
         })
-
         for number in range(1, 3):
             listed_hosts = HostCollection.hosts({
                 u'id': host_collection['id'],
-                u'organization-id': self.org['id'],
+                u'organization-id': self.organization['id'],
                 u'per-page': number
             })
             self.assertEqual(len(listed_hosts), number)
 
+    @tier2
+    def test_positive_copy_by_id(self):
```

Comment:
Why `copy`? Should be `clone` instead

Reply:
I suspect it's for correspondence with UI. But let's use our accepted naming conventions and maybe fix the naming for UI too instead of reusing wrong naming.

Reply:
command looks like 'hammer host-collection copy', so would like to keep it as it is